packages/mutation-testing-elements/testResources/simple-example/index.html

```diff
@@ -11,11 +11,11 @@
   <script>
     const app = document.getElementsByTagName('mutation-test-report-app').item(0)
```

Comment:
Maybe it makes sense to add the theme color as a parameter of the event? For example: `{ themeColor: "#18191a" }`? Otherwise, the application needs to guess what the color is.

Reply:
Yes I thought about that too and it's not too difficult, but the problem is when you first open the report no event is thrown to catch the initial background color

Reply:
Ah yes. Then it makes sense to also reflect this on the theme property here:

https://github.com/stryker-mutator/mutation-testing-elements/blob/d117561a83bf93a8d884d5fd6d84aa1c7918a4ae/packages/mutation-testing-elements/src/components/mutation-test-report-app/mutation-test-report-app.component.ts#L36-L37

That way you can either look at the property or look at the event data.

Reply:
Added `themeBackgroundColor` as a property